cime_config/namelist_definition_ufsatm.xml

```diff
@@ -9663,4 +9663,22 @@
     <value>postxconfig-NT.txt</value>
   </values>
 </entry>
+
+<!-- =========================== -->
+<!-- CIME internal -->
+<!-- =========================== -->
+
+<entry id="resolution_for_grib2">
+  <type>char</type>
```

Comment:
Please add a comment explaining why this is a char and not a real variable.

Reply:
Sure. I'll look at CIME side to fix the real issue, if I could I'll make change in here also. I am also setting s2s defaults now. Let's wait to test them before merge.